projects/m2/024-create-a-bingo-card/js/index.js

```diff
@@ -0,0 +1,32 @@
+function generateBingoCard() {
+    let ranges = [[1, 15], [16, 30], [31, 45], [46, 60], [61, 75]];
+    let headers = ['B', 'I', 'N', 'G', 'O'];
+    let bingoColumns = {};
```

Comment:
These **variables** are **declared incorrectly** (you always use `let` instead of `const`). 
Can you **fix** it?